### ISSUE TITLE
Improve Speed

### ISSUE THREAD

**Issue by OrkoHunter** [OrkoHunter/keep#38]
Profile the whole codebase and identify the bottlenecks.

**Comment by ioogithub** [comment]
I agree with this, when running "keep run" on a list with one saved command, there is a noticeable delay before it displays the list.

Speed is critical this type of app.
